### ISSUE TITLE
Unexpected redirect from 80 port

### ISSUE THREAD

**Issue by eduard93** [intersystems-ru/webterminal#50]
I'm opening the following page:`http://ip/terminal/` 
And receive the following output:
```
WebSocket connection error (ws://ip:57776/WebTerminal.Engine.cls). Trying to connect again in 10 seconds...

WebSocket connection closed. Code 1006, reason: .
```

So why 57776 port? 
Expected behavior: 
- if port is in url, connect to the same port
- if no port is present in url
  - if protocol is `http` connect to 80 port
  - if protocol is `https` connect to 443 port

**Comment by ZitRos** [comment]
Confirmed! Because of [this line](https://github.com/intersystems-ru/webterminal/blob/master/webSource/js/TerminalController.js#L31) in the project's sources: the output after build would be
```js
this.server = new CacheWebTerminalServer(
    this, (location.protocol === "https:" ? "wss:" : "ws:"), location.hostname,
    location.port || "57776"
  );
```
And the `location.port` can be equal to `""`.

Thanks, Edward.

**Comment by ZitRos** [comment]
Fixed by 46ad598dc1428575fdcccc7e53cb6509f0ee0189 ([3.2.2](http://intersystems-ru.github.io/webterminal/#downloads)), please, confirm problem solved and close the issue.
Wondering why nobody has never submit their issues about this.
